doc/doxygen/headers/coding_conventions.h

```diff
@@ -56,6 +56,14 @@ pre-commit hook. One way to do so, is to copy
 <code>\${SOURCE_DIR}/contrib/git-hooks/pre-commit</code> to
 <code>\${SOURCE_DIR}/.git/hooks/pre-commit</code> and make sure it is
 executable.
+
+On some systems the <code>clang-format</code> executable points to a more
+recent version such as <code>clang-format-10</code>. In this case you should
+replace the above <code>make indent</code> command with
+@code
+  make DEAL_II_CLANG_FORMAT=clang-format-6.0 indent
+@endcode
+provided <code>clang-format-6.0</code> is installed and in your path.
```

Comment:
There will rarely be a command that is actually named that way, but you probably mean that one can provide a full path to an executable with that specific version. Maybe replace `clang-format-6.0` by `/usr/local/clang-6.0/clang-format` in the two places you have it above, and say something like "...and assuming that you have installed version 6.0 of clang-format in /usr/local/clang-6.0. Of course, if you have installed that version of clang-format in a different directory, you should use the corresponding path."?

Reply:
Done.